content/en/posts/outer-misalignment.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Outer Misalignment"
+date: 2023-10-25T13:25:45-05:00
+summary: Outer misalignment is the mismatch between what humans intend an AI system to do and the objective that is actually given to the AI. 
+---
+
+Basic
+
+At its core, outer misalignment deals with the mismatch between what humans intend for an AI system to do and the objective that is actually given to the AI.
+Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you unintentionally teach it to fetch any paper-related item, including your important documents. This is a simple analogy for outer misalignment – the dog's action doesn't align with your true intentions.
```

Comment:
```suggestion
Imagine you're training a dog. You want the dog to fetch the newspaper for you, but instead of training it to do just that, you teach it to fetch any paper-related item, which could include your important documents. This is a simple analogy for outer misalignment – the instructions which you gave to the dog don't align with your true intentions.
```